app/Support/MaintenanceService.php

```
@@ -212,6 +212,17 @@ public function runAll(): array
             SecureLogger::error(__('MaintenanceService errore generazione ICS'), ['error' => $e->getMessage()]);
         }
 
+        // Purge expired / long-revoked "Remember me" sessions so user_sessions
+        // does not grow unbounded. This previously lived only in the orphaned
+        // scripts/maintenance.php, which no documented cron calls — so on a
+        // standard install expired remember-me tokens were never cleaned up.
+        try {
+            $results['expired_sessions_purged'] = (new RememberMeService($this->db))->cleanupExpiredSessions();
+        } catch (\Throwable $e) {
+            $results['errors'][] = 'cleanupExpiredSessions: ' . $e->getMessage();
+            SecureLogger::error(__('MaintenanceService errore pulizia sessioni scadute'), ['error' => $e->getMessage()]);
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'App\Support\SecureLogger' in method 'runAll'. (undefined)

(StaticAccess)